### PR TITLE
[fix](group commit) Group commit http stream should not begin txn

### PR DESCRIPTION
**Pull request by mymeiyi**
## Proposed changes

The http stream in group commit mode should not begin_txn, otherwise will hang schema change.

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

**Comment by mymeiyi** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by gavinchou — APPROVED** [review]

File: fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/OlapInsertExecutor.java [hunk 1]
```diff
@@ -147,13 +151,15 @@ public void finalizeSink(PlanFragment fragment, DataSink sink, PhysicalSink phys
...
             throw new AnalysisException(e.getMessage(), e);
         }
-        TransactionState state = Env.getCurrentGlobalTransactionMgr().getTransactionState(database.getId(), txnId);
-        if (state == null) {
-            throw new AnalysisException("txn does not exist: " + txnId);
-        }
-        addTableIndexes(state);
-        if (physicalOlapTableSink.isPartialUpdate()) {
-            state.setSchemaForPartialUpdate((OlapTable) table);
+        if (!isGroupCommitHttpStream()) {
```
Comment (gavinchou) [inline fe/fe-core/src/main/java/org/apache/doris/nereids/trees/plans/commands/insert/OlapInsertExecutor.java:154]: Try the following pattern to reduce indention and possible conflict
```suggestion
        if (isGroupCommitHttpStream()) {
            return;
        }
```

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by Yukang-Lian — APPROVED** [review]

**Review by gavinchou — APPROVED** [review]

**Review by gavinchou — APPROVED** [review]

**Pull request merged**